docs/live-queries.md

```diff
@@ -0,0 +1,1373 @@
+---
+title: Live Queries
+id: live-queries
+---
+
+# TanStack DB Live Queries
+
+TanStack DB provides a powerful, type-safe query system that allows you to fetch, filter, transform, and aggregate data from collections using a SQL-like fluent API. All queries are **live** by default, meaning they automatically update when the underlying data changes.
+
+The query system is built around a fluent API similar to SQL query builders like Kysely or Drizzle. The query builder doesn't perform operations in the order of method calls - instead, it composes your query into an optimal incremental pipeline that gets compiled and executed efficiently. Each method returns a new query builder, allowing you to chain operations together.
```

Comment:
Perhaps a hyperlink for fluent API -- I don't really know what a fluent api is. Or deprioritise the concept and say "built around a query builder API similar to ...".